packages/devtools_app/lib/src/shared/history_viewport.dart

```diff
@@ -61,13 +61,13 @@ class _HistoryViewportState<T> extends State<HistoryViewport<T?>> {
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
     return OutlineDecoration(
-      child: ValueListenableBuilder(
+      child: ValueListenableBuilder<T?>(
         valueListenable: widget.history.current,
-        builder: (context, dynamic current, _) {
+        builder: (context, T? current, _) {
           return Column(
             children: [
               _buildTitle(context, theme),
-              widget.contentBuilder(context, current),
+              widget.contentBuilder(context, current!),
```

Comment:
what if current is null? maybe we shouldn't be force unwrapping this here

Reply:
done